### ISSUE TITLE
unterscheidung user messages und unsere messages

### ISSUE THREAD

**Issue by cknott** [baehll/neosyn#104]
hi @baehll wie am telefon besprochen müssten alle nachrichten die von einer user interaktion auf social media kommen mit der jeweiligen user id im json response übermittelt werden. alle im nachrichtenverlauf von uns / einem nutzer von unaite.ai erstellten nachrichten sollten als user id den wert 0 übermitteln.

**Comment by baehll** [comment]
Messages unserer User haben nun "from": null im response, Kommentare von anderen haben die jeweilige ID dort stehen

**Comment by baehll** [comment]
implementiert in dev

**Comment by cknott** [comment]
cool, bisher sind alle nachrichten von unserem user, gibt es eine möglichkeit hier einen verlauf mit abwechselnden usern zu kriegen? muss ich mit meinem instagram account posts kommentieren?

**Comment by baehll** [comment]
ja, wenn du unter ein paar meiner posts kommentieren könntest, könnten wir ein paar kommentare mehr sehen

https://www.instagram.com/baehllord/

wenn du mich hinzufügst und mal wild rum kommentierst, wäre das super
